_topic_map.yml

```diff
@@ -763,6 +726,42 @@ Topics:
     File: monitoring-and-cluster-logging-operator-component-certificates
   - Name: Control plane certificates
     File: control-plane-certificates
+- Name: Compliance
```

Comment:
Should it be Compliance Operator?

Reply:
> I also moved the oc-compliance plugin assembly in with the compliance operator subfolder and renamed the subfolder to just "Compliance". Will need to get an ack on whether this is okay or not, and if so, if it needs a redirect

This is PR top comment. To clarify: so you have concern about the moving and renaming?

Reply:
"compliance" is a very big concept. It usually contains two part: "compliance checking" and "file integrity checking".
To be more accurate, maybe use "compliance-operator and oc-compliance plugin" instead.

Reply:
No worries, I can revert this change and update the folder name back to "Compliance Operator". That folder will still hold the oc-compliance plug-in doc though. But since it's related to the Compliance Operator, I think this is fine.

CC @JAORMX